localstack/services/awslambda/lambda_executors.py

```diff
@@ -699,10 +699,14 @@ class Util:
     @classmethod
     def get_java_opts(cls):
         opts = config.LAMBDA_JAVA_OPTS or ''
-        if '_debug_port_' in opts:
+        if 'address=_debug_port_' in opts:
             if not cls.debug_java_port:
                 cls.debug_java_port = get_free_tcp_port()
             opts = opts.replace('_debug_port_', ('%s' % cls.debug_java_port))
+        elif 'address=' in opts:
+            if not cls.debug_java_port:
+                cls.debug_java_port = opts.split('address=')[1]
```

Comment:
@giangvo I think we should use a regular expression here. What if `opts` looks like this:
```
-agentlib:jdwp=transport=server=y,address=1234,suspend=y
```
... then the port would get incorrectly parsed as `"1234,suspend=y"`. Let's also add this as a unit test, to avoid regressions in the future.

Reply:
I updated to use regex